docs/advanced/document-store/benchmark.md

```diff
@@ -197,17 +362,9 @@ If you're experimenting on a dataset with fewer than 10,000 Documents, you can u
 
 If your dataset does not fit in memory, and you **do not** care much about the speed of nearest neighbor search, you can use `sqlite` as storage.
```

Comment:
we still don't talk about other backends. I think we can share the following insight:
* Redis offers great speed if your data fits in memory and you need an isolated server to store your data